diff -up ./wx-config.in.multiarch ./wx-config.in
--- ./wx-config.in.multiarch	2013-10-13 19:34:37.890531217 +0300
+++ ./wx-config.in	2013-10-13 19:36:41.722535740 +0300
@@ -1286,7 +1286,7 @@ else
     _include_cppflags="-I${includedir}"
 fi
 
-_cppflags=`echo "-I${libdir}/wx/include/@TOOLCHAIN_FULLNAME@" $_include_cppflags "@WXCONFIG_CPPFLAGS@" $_gui_cppflags`
+_cppflags=`echo "-I${libdir}/wx/include/ -I${libdir}/wx/include/@TOOLCHAIN_FULLNAME@" $_include_cppflags "@WXCONFIG_CPPFLAGS@" $_gui_cppflags`
 
 # now without further ado, we can answer these too.
 [ -z "$output_option_cppflags" ] || echo $_cppflags
